modules/datadog-configuration/modules/datadog_keys/variables.tf

```diff
@@ -1,16 +1,5 @@
-variable "region" {
-  type        = string
-  description = "AWS Region"
-}
-
```

Comment:
this needs to be reverted, is used by several components in the reference to it:
e.g.
```hcl
module "datadog_configuration" {
  source  = "../datadog-configuration/modules/datadog_keys"
  region  = var.region
  context = module.this.context
}
```

Reply:
@Benbentwo I don't understand, `var.region` is not utilized anywhere in this `datadog_keys` module, so why does it need to be reverted?

Doesn't terraform only give a warning on passing an unused variable in to a child module? I think that's the correct way forward, then the warning is motivation to stop passing it because it is being unused.